_includes/field-description-state.md

```diff
@@ -0,0 +1,3 @@
+Indicates the state of the transaction, usually `initialized`, `completed` or
+`failed`. If a partial {{ transaction }} has been done and further transactions
+are possible, the state will be `awaitingActivity`.  
```

Comment:
I think `_includes/field-description-state.md` needs to take greater care of linefeeds. Checkout out how this is done in `field-description-id.md` for instance.

Reply:
@arebra ☝️

Reply:
Is this what you meant?

Reply:
Almost! :)